tck/src/main/java/org/eclipse/microprofile/config/tck/util/AdditionalAssertions.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
```

Comment:
2017, 2021
Since you are refactoring the code instead of writing from scratch, you need to use the original copyright year and copy the authors from the original file as well.